pkg/antctl/antctl.go

```diff
@@ -76,6 +80,19 @@ var CommandList = &commandList{
 			CommandGroup:        flat,
 			AddonTransform:      versionTransform,
 		},
+		{
+			Use:                 "agent-info",
+			Short:               "Print agent's information",
+			Long:                "Print agent's information of the antctl.",
```

Comment:
What does "of the antctl" mean here? Maybe we can come up with a better help message.

Reply:
Updated the message.